### PR TITLE
Add boards command

### PR DESCRIPTION
**Pull request by Gawetaner**
Adds the command `boards` that opens the boards of a gitlab project.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -227,6 +227,26 @@ function browse(options) {
...
 
+function boards(options) {
+  logger = log.getInstance(options.verbose);
+
+  getBaseBranchName().then(function (curBranchName) {
+    getRemoteForBranch(curBranchName).then(function (remote) {
+      if (!remote) {
+        console.error(colors.red('Branch ' + curBranchName + ' is not tracked by any remote branch.'));
+        console.log('Set the remote tracking by `git remote git branch --set-upstream <branch-name> <remote-name>/<branch-name>`');
+        console.log('Eg: `git branch --set-upstream foo upstream/foo`');
```
Comment (vishwanatharondekar) [inline index.js]: Shouldn't there be `process.exit(1)` here. You will want to stop the process with error when getting the remote has failed.

Comment (Gawetaner) [reply index.js]: I just adopted the implementation of `browse` command. I can add a separate pull request to fix it in both commands, if this one will be accepted.

**Comment by vishwanatharondekar** [comment]
On a separate point there are lot many URLs like these which can be supported out of the box by just appending the command. We can reuse the same code in all the commands. What do you think about that.

**Comment by Gawetaner** [comment]
That was my first idea, but for a first draft i just added the command.

I will try to add the options like `labels`, `boards`, `milestones` to the browse command.\
What do you think, add them explicit by handling each option seperatly or implicit by just adding the option specified by the user to the path?

`open(gitlab.options.url + '/' + projectName + '/boards/');`

or 

`open(gitlab.options.url + '/' + projectName + option);`

**Comment by vishwanatharondekar** [comment]
Adding whitelist is unnecessary. It's not going to add any value. 

If user does put something which is not present he will get a 404 page and should understand that he made a mistake. The browse command is shortcut utility anyway. Adding this option is just making it more powerful. 

Keeping it generic will keep the code changes to minimum and give flexibility of creating future URLs as well. Just add an option which will be appended to URL. If not present should just open project page.

**Comment by Gawetaner** [comment]
I removed the `boards` command and added an option to the `browse` that allows to open a specific page in gitlab.

**Pull request merged**